# Request
---
# Response
float32[6] errorvalue
bool success
